config.go

```diff
@@ -41,14 +41,26 @@ const (
 	_packageName       = "yarpc"
 )
 
+// LogLevelConfig configures the levels at which YARPC logs various things.
+type LogLevelConfig struct {
+	// Level at which application errors are logged.
```

Comment:
I realize that it breaks the abstraction a bit, but can we explicitly document that all Thrift exceptions are application errors?